tbx/core/blocks.py

```diff
@@ -100,6 +102,28 @@ class Meta:
 class PullQuoteBlock(StructBlock):
```

Comment:
**Issue**: the role / company field is missing here.

Reply:
Nice spot! I've added a note to ticket to summarise the discussion in stand-up about this where we agreed to include a new ticket to make sure that imported quotes are nicely formatted without their image / role but that our preference is to keep these new fields as required